Development

Ankit-Nag opened the pull request.

Ankit-Nag committed:
pacifica v0

Ankit-Nag committed:
fix nav link

Fix: Nav github link updated

Ankit-Nag committed:
update comments

added/updated comments for better readability

Ankit-Nag committed:
Update README.md

Ankit-Nag committed:
Update README.md

Ankit-Nag committed:
style update

style:

1.change font-styles

2. minor styling changes: code snippets, padding, font-weights

omkarborude reviewed (approved):
Nice work Ankit, I also learned from this Code!

harshitbadollacodes reviewed (commented):
Looks great. Keep adding new components 👍

Ankit-Nag committed:
update button styles

style: removed box shadow on hover from buttons

Ankit-Nag committed:
Update styles.css

changed outline buttons to be more compliant with dark mode

Ankit-Nag committed:
style, documentation update

docsite: Site is now responsive excluding iframes.

scrollbar: fixed horizontal scroll bar
